src/main/java/Command.java

```diff
@@ -0,0 +1,100 @@
+import java.util.ArrayList;
+
+public abstract class Command {
+    public abstract boolean execute(Storage storageController, ArrayList<Task> storage);
+}
+
+class AddCommand extends Command {
```

Comment:
You could abstract all these various commands into their own Java classes that inherit/implement `Command.java`, if you want even `A-MoreOOP` (e.g. `AddCommand.java`, `DeleteCommand,java`, etc.)!

Reply:
Was keeping them together at the beginning so I can see the classes more easily. I have moved all the subclasses of `Command`, as well as `Task` to separate files.